spec/spec_helper.rb

```diff
@@ -30,7 +33,7 @@
     #     # => "be bigger than 2"
     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
   end
-
+  
```

Comment:
Admittedly I used to do this all the time, OCD and all, but yeah the merge conflicts avoidance is worth it.